src/js/containers/generateFiles/GenerateFilesContainer.jsx

```diff
@@ -56,7 +56,8 @@ class GenerateFilesContainer extends React.Component {
                     header: '',
                     description: ''
                 },
-                showDownload: false
+                showDownload: false,
+                fundingAgency: false
             },
```

Comment:
Recommend changing state property `d1/d2.fundingAgency` to `d1/d2.isFundingAgency` b/c it's a boolean and prepending `is` makes that a bit more readable IMO.

Reply:
Agreed